### ISSUE TITLE
[Meta] Plan :)

### ISSUE THREAD

**Issue by JackKelly** [openclimatefix/power_perceiver#23]
- [ ] Create a `SelectPVSystemsWithinImage` xr_batch_processor (i.e. split this out from the existing `Select1PVSystem` xr_batch_processor) which maybe just drops any examples which don't have any PV systems in the region of interest.
- [ ]

**Comment by peterdudfield** [comment]
Would it be worth trying putting `elevation` in too? 

Im not sure where this where this goes on the priority list, perhaps lower down

**Comment by JackKelly** [comment]
Good idea! Thanks! After your suggestion, I've just added `elevation` after `include land/sea mask` :slightly_smiling_face:
